gcp_variant_transforms/transforms/write_variants_to_shards.py

```diff
@@ -37,7 +37,11 @@ def __init__(self, vcf_shards_output_dir, number_of_variants_per_shard):
     """
     self._vcf_shards_output_dir = vcf_shards_output_dir
     self._number_of_variants_per_shard = number_of_variants_per_shard
-    self._coder = vcfio._ToVcfRecordCoder()
+    # Write shards as is.
```

Comment:
nit: Could you please mention this is related to `_shard_variants()` where it calls `_read_variants()` with `use_1_based_coordinate=False`.

Reply:
Hmmm, but I already have it below...
I'll make the comment a bit more explicit.